packages/moertel/src/mortar/mrtr_overlap_Def.hpp

```diff
@@ -381,8 +381,32 @@ bool MOERTEL::Overlap<IFace>::build_sxim()
   double gap;
   for (int i=0; i<nsnode; ++i)
   {
+
+    {
+      const double * node_norm = snode[i]->Normal();
```

Comment:
This loop body has an awkward mix of C++ and C data structures, but it must be because you're changing existing code.

Reply:
I will try to change that loop body.